src/test/ui/type-alias-impl-trait/cross_crate_ice.rs

```diff
@@ -1,9 +1,9 @@
 // aux-build:cross_crate_ice.rs
-// build-pass (FIXME(62277): could be check-pass?)
```

Comment:
Rejecting extern opaque type aliases is kind of confusing. I wasn't able to create good diagnostics for this case.

Reply:
For an initial implementation, the diagnostic here for extern opaque type aliases is good enough.